<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QWidget" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入要修改密码的账号、手机号、邮箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangePushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>570</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackPushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="IDLable">
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="5">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="IDLineEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="PhoneLineEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="EmailLineEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="PasswordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="AgainPasswordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="PhoneLable">
      <property name="text">
       <string>手机号</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>邮箱</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>请输入密码</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>再次输入密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
